python/ray/train/backend.py

```diff
@@ -34,6 +35,11 @@ class BackendConfig:
     def backend_cls(self):
         raise NotImplementedError
 
+    @property
+    def accelerator_cls(self) -> Optional[Type[Accelerator]]:
+        """The ``Accelerator`` type for this backend, if any."""
```

Comment:
To clarify, I'm suggesting to remove the `accelerator_cls` concept.

Reply:
This is just for constructing a default accelerator right? Does it prevent the user from instantiating a different one?

Reply:
Yeah this is for a "default accelerator", but we can implement this without having to tie it to the backend.

I just feel that if we keep this more modular, instead of tying the components together then it sets us up for refactoring if needed down the line.

Reply:
Sorry for missing this but what's the way to implement it without tying it to the backend?

Reply:
In `train.torch.prepare_model`, `train.torch.prepare_data_loader` we can instantiate a `TorchAccelerator` if one doesn't exist.

Reply:
But if we make the Accelerator class the user facing API, then the idea of a default accelerator is a moot point anyways. You have to instantiate it before calling any methods on it.

Reply:
Ah yeah I'm okay with that - this could go in the proposed `get_current_accelerator()`?

Reply:
yep!